### ISSUE TITLE
Autoplaying animations in violations of IBM Checkpoint 2.2.2

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/design-language-website#411]
The autoplaying videos and animations are in violations of [Checkpoint 2.2.2](https://www.ibm.com/able/guidelines/ci162/pause_stop_hide.html). The videos can be right-clicked and paused, but on screen, keyboard accessible controls should be made available.

**Comment by jnm2377** [comment]
@dakahn are you referring to the homepage video? If not, I'm not sure what other autoplaying videos the IDL site has. Could you provide more info.

**Comment by vpicone** [comment]
@jnm2377 this is actually for the animations in the cards. If I understand correctly, the start/stop/pause requirement means anything that plays for longer than 5 seconds is a violation. So we need to either:

1. change all our gifs to be only 5 seconds
2. add a start/stop/pause ui to the gifs.

**Comment by dakahn** [comment]
@jnm2377 Correct! The embedded videos found in the cards on the homepage as well as in the Galleries section are what I'm referring to.

**Comment by jnm2377** [comment]
ahh, gotcha! wondering if that's something that we should add on GTC? Like a custom component that will handle pause/play for gifs? As I imagine this would also apply to any gif in any of our sites. @vpicone

**Comment by jnm2377** [comment]
update: gif player component has been merged into GTC. Once we update the latest version to include this commit, we can open a PR in IDL and carbon website to update any existing gifs to use this component.

**Comment by jnm2377** [comment]
update: gtc has been updated so the latest version now includes the gif player.
these are the pages that will need to be migrated in idl:

- homepage https://www.ibm.com/design/language/
    - feature card “elements Photography” - 1 gif
- Gallery page: https://www.ibm.com/design/language/gallery
    - 12 gifs total
- Color page: https://www.ibm.com/design/language/color
    - gradients section - 1 gif
    - color in ui section - 1 gif

We need static images for all of these gifs.
